packages/phone-number-privacy/combiner/src/database/wrappers/account.ts

```diff
@@ -39,27 +82,44 @@ export async function getDidMatchmaking(account: string, logger: Logger): Promis
 /*
  * Set did matchmaking to true in database.  If record doesn't exist, create one.
  */
-export async function setDidMatchmaking(account: string, logger: Logger) {
+export async function setDidMatchmaking(
+  account: string,
+  logger: Logger,
+  signedUserPhoneNumber?: string
+) {
   logger.debug({ account }, 'Setting did matchmaking')
-  try {
-    if (await getAccountExists(account)) {
+  if (await getAccountExists(account, logger)) {
+    try {
```

Comment:
Maybe something like ?
```
const query = accounts()
          .where(ACCOUNTS_COLUMNS.address, account)
          .update(ACCOUNTS_COLUMNS.didMatchmaking, new Date())
if(!!signedUserPhoneNumber) {
        query.update(ACCOUNTS_COLUMNS.signedUserPhoneNumber, signedUserPhoneNumber)
}
return query().timeout(DB_TIMEOUT)
```

Let build queries with query builder.

Reply:
(never used knex though so make sure that we don't run the query 3 times because of this code xD)